adapter/comment_adapter_test.go

```diff
@@ -0,0 +1,103 @@
+package adapter
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/nibble-4bits/ondemand-go-bootcamp/entity"
+	"github.com/nibble-4bits/ondemand-go-bootcamp/httpClient"
+
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
Great, you are using the convention to import libraries:

// core libraries

// our libraries

// third party library